### PR TITLE
Update kolibri-tools for external plugin building.

### LINKED ISSUE #5865: update perseus to use new build config scheme
### Observed behavior

follow-up from #5864, need to update perseus to use new buildconfig.  Currently builds but does not run.

### Errors and logs

Currently getting:

```
ERROR    Internal Server Error: /en/user/
Traceback (most recent call last):
  File "/Users/d/Projects/le/kolibri/kolibri/core/webpack/hooks.py", line 111, in _stats_file_content
    with io.open(self._stats_file, mode="r", encoding="utf-8") as f:
FileNotFoundError: [Errno 2] No such file or directory: '/Users/d/Projects/le/kolibri/.venv/lib/python3.7/site-packages/kolibri_exercise_perseus_plugin/build/_stats.json'
```

### Context

current 0.13.0 develop branch

### PR DESCRIPTION
**Pull request by rtibbles**
### Summary
Small tweaks to plugin building.
Upgrade perseus renderer with new build configuration.

### Reviewer Guidance
Does the devserver run and load pages without errors.

### References
Fixes #5865 

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [x] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by indirectlylit** [comment]
changes: https://github.com/learningequality/kolibri-exercise-perseus-plugin/compare/68c9decf1940ee3d806c0c8c6659cb100bda0041...0cfdfebfc6360271b1f787727a03d7972194979e

File: packages/kolibri-tools/lib/webpack_json.py [hunk 1]
```diff
@@ -15,6 +15,9 @@
 
 logger = logging.getLogger("webpack_json")
 logger.setLevel(level=logging.INFO)
+ch = logging.StreamHandler()
+ch.setLevel(logging.INFO)
+logger.addHandler(ch)
```
Comment (indirectlylit) [inline packages/kolibri-tools/lib/webpack_json.py]: what does 'ch' stand for?

File: packages/kolibri-tools/lib/webpack.config.base.js [hunk 2]
```diff
@@ -48,7 +48,32 @@ module.exports = (data, { mode = 'development', hot = false } = {}) => {
     logging.error(data.name + ' plugin is misconfigured, missing parameter(s)');
     return;
   }
-
+  const configData = require(data.config_path);
```
Comment (indirectlylit) [inline packages/kolibri-tools/lib/webpack.config.base.js:51]: nice work-around

File: packages/kolibri-tools/lib/webpack_json.py (hunk 1 above)
Comment (rtibbles) [reply packages/kolibri-tools/lib/webpack_json.py]: It stands for: "Richard copied this from the Python docs" - https://docs.python.org/3/howto/logging-cookbook.html#using-logging-in-multiple-modules

**Review by indirectlylit — APPROVED** [review]
this works well for me.

File: packages/kolibri-tools/lib/webpack_json.py (hunk 1 above)
Comment (indirectlylit) [reply packages/kolibri-tools/lib/webpack_json.py]: super minor, recommend updating the variable name before merging :P

Comment (rtibbles) [reply packages/kolibri-tools/lib/webpack_json.py]: To be honest, I'm not even sure this fixes the logging problem.

**Pull request merged**